test/unit/math/mix/prob/std_normal_test.cpp

```diff
@@ -0,0 +1,48 @@
+#include <stan/math/mix/scal.hpp>
+#include <test/unit/math/rev/scal/fun/util.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+std::vector<double> test_fun(double y) {
```

Comment:
All the tests in this file can be replaced with something like:

```
auto f = [](const auto& y) { return stan::math::std_normal_lpdf(y); };
  
stan::test::expect_ad(f, -0.3);
stan::test::expect_ad(f, 0.0);
stan::test::expect_ad(f, 1.7);
```

Reply:
Cool!